Remove CXXFLAGS, CFLAGS, LDCLAGS set in configure.ac and Makefile.am
Written and tested by Lucas Mitrak

--- SEECER-0.1.3/Makefile.am
+++ SEECER-0.1.3/Makefile.am
@@ -1,6 +1,6 @@
 AM_LDFLAGS = -lgomp
 AM_CPPFLAGS = -Wno-deprecated -Wall $(SEQAN_CFLAGS) -fopenmp
-AM_CXXFLAGS = -O3 -g
+AM_CXXFLAGS =
 
 bin_PROGRAMS = bin/seecer \
 	bin/random_sub_N \
--- SEECER-0.1.3/configure.ac
+++ SEECER-0.1.3/configure.ac
@@ -2,10 +2,10 @@ AC_INIT([seecer], [1.0], [hple@cs.cmu.edu,maschulz@cs.cmu.edu])
 AM_INIT_AUTOMAKE([-Wall subdir-objects  foreign])
 AC_PREREQ(2.62)
 
-: ${CXXFLAGS=""}
-CFLAGS="$CFLAGS -I/opt/local/include"
-CXXFLAGS="$CXXFLAGS -I/opt/local/include"
-LDFLAGS="$LDFLAGS -L/opt/local/lib"
+#: ${CXXFLAGS=""}
+#CFLAGS="$CFLAGS -I/opt/local/include"
+#CXXFLAGS="$CXXFLAGS -I/opt/local/include"
+#LDFLAGS="$LDFLAGS -L/opt/local/lib"
 AC_PROG_CXX
 
 AC_CONFIG_HEADERS([config.h])
